### PR TITLE
Create smart contract for Amphotericin B Liposomal prescription

### LINKED ISSUE #44: Create smart contract for Amphotericin B Liposomal prescription
ℹ️ This issue is part of Epic #43 

## Objective

Create the smart contract for Amphotericin B Liposomal prescription for Leishmaniasis treatment

## Checklist

- [ ] The smart contract calculates the correct dosage for Amphotericin B Liposomal considering the new requirements of the physician documented in [Project Documents](https://www.notion.so/kommit/Cambios-propuestos-para-la-nueva-versi-n-edce0a10451542488473e845708be78a#1d20f71961274cfaacf9bef1be3371dd) pages 2 - 9.
- [ ] There is a formulation file which is called from the smart contract 
- [ ] There is a function to call the smart contract in the file `okdose/transmissionTypes/transmittedByVectors/leishmaniasis/index.js`
- [ ] The smart contract has been deployed to the default dev account
- [ ] There is a unit test for the smart contract

### PR DESCRIPTION
**Pull request by keliumJU**
## Description

Add translation and connection for the smart contract for Amphotericin B Liposomal prescription for Leishmaniasis treatment

Resolves #44 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: okdoseweb/assets/locales/en/translation.json [hunk 1]
```diff
@@ -60,6 +62,30 @@
         "28 days"
       ],
       "note": "Dosage divided into 2 to 3 daily doses"
+    },
+    "amphotericin_b_liposomal": {
+      "name": "Amphotericin B Liposomal",
+      "presentation": ["AmBisome® - Blister vial x 50 mg", "{{dose}} $t(dosage_unit)", "immunocompetent patient", "immunocompromised patient"],
```
Comment (LauraVilladakommit) [inline okdoseweb/assets/locales/en/translation.json:68]: @keliumJU The doctor had requested the change of "AmBisome®" for "Amphotericin B Liposomal". Please check the information here: https://www.notion.so/kommit/Cambios-propuestos-para-la-nueva-versi-n-edce0a10451542488473e845708be78a. For Miltefosine he asked to remove the word "Impavido" in the presentation. Can you add also this change, please?

File: okdose/transmissionTypes/transmittedByVectors/leishmaniasis/index.js [hunk 2]
```diff
@@ -14,9 +14,14 @@ const prescribeMiltefosine = async (nearInterface, weight) => (
   nearInterface.viewMethod(process.env.MILTEFOSINE_CONTRACT, process.env.CONTRACT_METHOD, { weight })
 );
 
+const prescribeAmphotericinBLiposomal = async (nearInterface, weight, prescriptionType) => (
```
Comment (jcamilocp) [inline okdose/transmissionTypes/transmittedByVectors/leishmaniasis/index.js:17]: Reading this line, I think we will need to document these functions. Maybe we can open a new issue for that @LauraVilladakommit, and after this, we need to add that item to every issue we create.

File: okdoseweb/assets/locales/en/translation.json [hunk 3]
```diff
@@ -52,14 +54,38 @@
...
+      "dosage_prescription": "{{volumeDose}} a day, infusion during 2 hours ({{weightDose}})",
+      "warning": "Do not exceed 250 mg per day",
+      "treatment_time": {
+        "lv_3mg": "Days 1 to 5 and then on days 14 and 21",
+        "lv_4mg": "Days 1 to 5 and then on days 10, 17, 24, 31 and 38",
+        "lcm_2mg": "10 to 20 days",
+        "lcm_3mg": "7 to 13 days"
+      },
+      "preparation": [
+        "Reconstitute each ampoule of AmBisome® with 12 mL of sterile water for injections",
```
Comment (LauraVilladakommit) [inline okdoseweb/assets/locales/en/translation.json:78]: @keliumJU remember to check the commercial name in the preparation field please, same in Miltefosine if apply.

**Review by LauraVilladakommit — APPROVED** [review]

**Review by jcamilocp — APPROVED** [review]
LGTM

**Pull request merged**